plugins/wasi_llm/wasillmfunc.cpp

```diff
@@ -13,68 +14,94 @@ namespace Host {
 Expect<WASILLM::ErrNo>
 WasiLLMModelCreate::bodyImpl(const Runtime::CallingFrame &Frame,
                              uint32_t CheckPointPath,
-                             uint32_t CheckPointPathLen) {
-  (void)Frame;
-  (void)CheckPointPath;
-  (void)CheckPointPathLen;
-  return WASILLM::ErrNo::InvalidArgument;
-}
+                             uint32_t CheckPointPathLen, uint32_t ModelIdPtr) {
+  auto *MemInst = Frame.getMemoryByIndex(0);
+  if (MemInst == nullptr) {
+    return Unexpect(ErrCode::Value::HostFuncError);
+  }
+  auto CheckPointPathSpan =
+      MemInst->getSpan<char>(CheckPointPath, CheckPointPathLen);
+  if (unlikely(CheckPointPathSpan.size() != CheckPointPathLen)) {
+    return WASILLM::ErrNo::InvalidArgument;
```

Comment:
It would be better if we could use `spdlog` to print some errors when encountering some kinds of errors.

Reply:
done